docs/features/dpdk/linux-networking.md

```diff
@@ -0,0 +1,431 @@
+# Linux Networking
+
+<!--
+Copyright 2022-2023 Intel Corporation
+SPDX-License-Identifier: Apache-2.0
+-->
+
+This page describes a step-by-step procedure to run Linux networking scenario.
```

Comment:
This kind of comment is appropriate for a standalone document. It's not appropriate for a user guide.

For the user guide, the initial paragraph(s) should introduce the reader to the **feature**. The page/chapter/document are rarely, if ever, mentioned.

"The Linux Networking feature ..."